src/REAXFF/fix_qeq_reaxff.cpp

```diff
@@ -360,7 +361,10 @@ void FixQEqReaxFF::allocate_matrix()
     i = ilist[ii];
     m += numneigh[i];
   }
-  m_cap = MAX((int)(m * safezone), mincap * REAX_MIN_NBRS);
+  bigint m_cap_big = MAX((int)(m * safezone), mincap * REAX_MIN_NBRS);
```

Comment:
```suggestion
  bigint m_cap_big = MAX((bigint)(m * safezone), (bigint)mincap * REAX_MIN_NBRS);
```
Same as in fix_qeq.cpp